src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -352,6 +353,7 @@ export class ResultsViewPageStore {
             cosmicCount: false,
             cosmicCountThreshold: 0,
             driverFilter: false,
+            selectedDriverTiers: false,
```

Comment:
why do we need this on top of the driverTiers object?

Reply:
@oplantalech can you respond to this here?

Reply:
@alisman I just fixed it, and made a getter instead